android/app/src/main/kotlin/coop/polypoly/polypod/OnboardingActivity.kt

```diff
@@ -17,7 +29,7 @@ class OnboardingActivity : AppCompatActivity() {
             close()
         }
         val carousel = findViewById<CarouselView>(R.id.carousel)
-        val strings = listOf(
+        var strings = mutableListOf(
```

Comment:
Is there a why to DRY this so that you don't need to add 4 lines of code per slide?

Reply:
Maybe later, right now we have some time pressure to finish this :D It got us this far I guess.